src/Tests/NServiceBus.Encryption.MessageProperty.Tests.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472;net6.0;net7.0</TargetFrameworks>
+    <TargetFramework>net8.0</TargetFramework>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
     <Optimize>false</Optimize>
```

Comment:
@bording @kentdr I've seen a few of those, do we need them?

Reply:
We should remove `<Optimize>false</Optimize>`. I'm not aware of any reason to have that in test projects at this point.

Reply:
I removed it.